main.html

```diff
@@ -250,6 +250,9 @@
             <div>
                 <span i18n="statusbar_cycle_time"></span> <span class="cycle-time">0</span>
             </div>
+            <div>
+                <span class="cpu-load">CPU Load: 0%</span>
```

Comment:
I think this should be using `i18n="statusbar_cpu_load"` that you are defining in `messages.json` above.